msbuild/Xamarin.MacDev.Tasks.Core/LinkerOptions.cs

```diff
@@ -55,6 +56,20 @@ public void BuildNativeReferenceFlags (TaskLoggingHelper Log, ITaskItem[] Native
 					if (Path.GetExtension (path) != ".framework")
 						path = Path.GetDirectoryName (path);
 					Frameworks.Add (path);
+				} else if (kind == NativeReferenceKind.Dynamic) {
```

Comment:
Should this be behind a platform being targeted check, so people don't use it on iOS and then later fail/complain on submission, or is letting them bounce on submission fine?

Reply:
I think that letting them bounce on submission is fine (after all dylibs work fine during development, when it can be useful).